docs/pages/operations-guide.md

```diff
@@ -252,6 +252,38 @@ Configuring `ROOT` vs `MOUNT` volumes may depend on the service. Some services w
 
 A Task generally maps to a process. A Pod is a collection of Tasks that share an environment. All Tasks in a Pod will come up and go down together. Therefore, [restart](#restart-a-pod) and [replace](#replace-a-pod) operations are at Pod granularity rather than Task granularity.
 
+## Overlay networks
+
+The SDK supports `pods` joining the `dcos` overlay network. You can specify that a pod should join the overlay by adding the following to your service spec YAML:
```

Comment:
The SDK allows `pods` to join the `dcos` overlay network. Specify that a pod should join the overlay by adding the following to your service spec YAML: